.gitignore

```diff
@@ -1,4 +1,11 @@
 .rts2_cache*
 .idea
 dist/
-node_modules/
\ No newline at end of file
+.test-space/
+coverage/
+node_modules/
+.DS_Store
+
+# temporary ignored during raid => choose one to keep afterwards
+package-lock.json
+yarn.lock
```

Comment:
@smeijer what do you want to do here? Do you have a preference or would you like me to disable lock files all together and not treat this as temporary?

Reply:
I don't really hold a strong opinion on this. This seems fine to me.

Reply:
We can remove comment and add config to disable lock files by default.

Reply:
Sounds good to me.